Extensions/PopUp.json

```diff
@@ -0,0 +1,282 @@
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/message-alert.svg",
+  "shortDescription": "Display pop-ups to alert, ask confirmation, and let user type a response in text box.",
+  "version": "1.0.0",
+  "tags": [
+    "pop up",
+    "alert",
+    "message",
+    "prompt",
+    "confirm",
+    "dialog"
+  ],
+  "authorIds": [
+    "HBp2oDKV87gKFbdd1wyV1jRYRC13"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "description": "The response to a pop-up message is valid or not empty. ",
+      "fullName": "Valid response",
+      "functionType": "Condition",
+      "group": "",
+      "name": "PromptValidResponse",
+      "private": false,
+      "sentence": "Response from the pop-up message with identifier: _PARAM1_ is valid or not empty",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "eventsFunctionContext.returnValue = runtimeScene.getGame().getVariables().get(eventsFunctionContext.getArgument(\"ID\")).getAsBoolean()\n",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Pop-up identifier",
+          "longDescription": "",
+          "name": "ID",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Return the text response by user to prompt.",
+      "fullName": "Response to prompt",
+      "functionType": "StringExpression",
+      "group": "",
+      "name": "PromptResponse",
+      "private": false,
+      "sentence": "Open prompt with message : _PARAM1_ with ID: _PARAM2_",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "const id = eventsFunctionContext.getArgument(\"ID\");\r\n\r\neventsFunctionContext.returnValue = runtimeScene.getGame().getVariables().get(id).getAsString();\r\n",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Pop-up identifier",
+          "longDescription": "",
+          "name": "ID",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Displays a prompt in a pop-up that prompts the user for input. This action return the text input or the false boolean if canceled.",
+      "fullName": "Prompt",
+      "functionType": "Action",
+      "group": "Web browsers",
+      "name": "Prompt",
```

Comment:
Like confirm, make this an expression directly. If null, return an empty string.